src/Render/Markdown/TemplateRenderer.cs

```diff
@@ -1,10 +1,21 @@
 ﻿using Scriban;
+using System.IO;
+using System.Reflection;
 using System.Threading.Tasks;
 
 namespace D2L.Dev.Docs.Render.Markdown {
 	internal class TemplateRenderer {
 		private readonly Template m_template;
-		
+
+		public static TemplateRenderer Create( string resourceName ) {
+			resourceName = resourceName.Replace( "/", "." );
+			var assembly = Assembly.GetExecutingAssembly();
+			using var stream = assembly.GetManifestResourceStream( $"{assembly.GetName().Name}.{resourceName}" )
+				?? throw new System.NullReferenceException( "Could not load resource " + $"{assembly.GetName().Name}.{resourceName}" );

```

Comment:
Is this throw necessary?

Reply:
I guess not? It provides a bit more detail on an error, but if we are providing resource names then I guess this shouldn't happen.

Reply:
Yeah I think just having it crash implicitly is fine. A bug here wouldn't be caused by user input, but by us just totally breaking the exe.